manim/scene/graph_scene.py

```diff
@@ -515,11 +515,18 @@ def get_riemann_rectangles(
                 y_point = 0
             else:
                 y_point = bounded_graph.underlying_function(x)
-            points = VGroup(*list(map(VectorizedPoint, [
-                self.coords_to_point(x, y_point),
-                self.coords_to_point(x + width_scale_factor * dx, y_point),
-                graph_point
-            ])))
+            points = VGroup(
+                *list(
+                    map(
+                        VectorizedPoint,
+                        [
+                            self.coords_to_point(x, y_point),
+                            self.coords_to_point(x + width_scale_factor * dx, y_point),
+                            graph_point,
+                        ],
+                    )
+                )
+            )
```

Comment:
Ok I can take the commas, but this is pretty ridiculous to me. @eulertour your thoughts? Can we make black behave here?